pkg/servicecentral/metadata.go

```diff
@@ -29,6 +29,16 @@ func SetPagerDutyMetadata(serviceCentralData *ServiceDataWrite, m *creator_v1.Pa
 	return setMetadata(serviceCentralData, PagerDutyMetadataKey, m)
 }
 
+// GetOpsGenieAttribute reads the Opsgenie team attribute out of a service
+func GetOpsGenieAttribute(serviceCentralData *ServiceDataRead) (*creator_v1.OpsgenieMetadata, error) {
```

Comment:
Capital `G` =)

Reply:
I wonder why do we have all these functions exported here?

Reply:
Good question, looks like they are all used within the same package. I'll fix these in a follow up PR.